_includes/sidebar.html

```diff
@@ -19,7 +19,7 @@ <h3>How to apply this license</h3>
 
   {% if page.source %}
```

Comment:
@mlinksva this will now never be true. We'll need to remove this conditional if we want the source link to display.